app/utils/callback/lava.py

```diff
@@ -43,8 +44,7 @@
 
 TEST_CASE_MAP = {
     models.NAME_KEY: "name",
-    models.STATUS_KEY: "result",
-    models.INDEX_KEY: "logged",
+    models.STATUS_KEY: "result"
```

Comment:
It's better to keep the trailing comma, to reduce the diff noise.

Reply:
fixed